packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* Create camera_android_camerax plugin for development.
```

Comment:
nit

```suggestion
* Creates camera_android_camerax plugin for development.
```

Reply:
And now that I think about it. This may change depending on whether we are publishing now or are waiting until the code is finished.